src/bindings/c/docs/api_overview.md

```diff
@@ -449,7 +449,7 @@ This strcut represents OpenVINO entity and allows you to manipulate with plugins
 
 - `ov_status_e ov_core_read_model(const ov_core_t* core, const char* model_path, const char* bin_path, ov_model_t** model)`
 
-  - Description: Reads models from IR / ONNX / PDPD / TF / TFLite formats to create ov_model_t.
+  - Description: Reads models from IR / PyTorch / ONNX / PDPD / TF / TFLite formats to create ov_model_t.
```

Comment:
we don't support reading of PyTorch formats. We can only convert model (Python object) from PyTorch via convert_model

Reply:
ok, remove it.

Reply:
My bad)